geomesa-gs-monitor-elasticsearch/src/test/scala/org/geomesa/gs/monitor/elastic/ExtendedRequestDataTest.scala

```diff
@@ -399,10 +398,10 @@ class ExtendedRequestDataTest extends Specification {
       rd.setStatus(status)
       rd.setTotalTime(totalTime)
 
-      val erd = new ExtendedRequestData(rd)
+      val erd = new ExtendedRequestData(rd, mockCatalog)
 
       // null fields should be excluded regardless
-      val excludedFields = Set("httpMethod", "responseLength", "responseContentType", "subOperation", "remoteLat", "remoteLon", "id", "internalid")
+      val excludedFields = Set("httpMethod", "responseLength", "responseContentType", "subOperation", "remoteLat", "remoteLon", "id", "internalid", "catalog")
```

Comment:
ah, does 'catalog' end up getting serialized in the json?

Reply:
That was more of a precautionary measure, but I think it would otherwise try to. I initially had it as `private val catalog: Catalog` in the constructor thinking that would prevent it from trying to serialize, but nothing was being sent to elasticsearch. Once I made it public and told gson to ignore it everything came through.

Reply:
this is a test case, is it excluded in src/main too?